src/ucp/rndv/rndv_get.c

```diff
@@ -207,6 +207,23 @@ static void ucp_rndv_get_zcopy_proto_abort(ucp_request_t *request,
     }
 }
 
+static void ucp_rndv_get_zcopy_proto_reset(ucp_request_t *request)
+{
+    switch (request->send.proto_stage) {
+    case UCP_PROTO_RNDV_GET_STAGE_FETCH:
+        ucp_proto_request_zcopy_reset(request);
+        break;
+    case UCP_PROTO_RNDV_GET_STAGE_ATS:
+        ucp_proto_request_bcopy_reset(request);
```

Comment:
looks weird that we call "bcopy" reset from get_zcopy_proto_reset
maybe break to complements with "fall-through" ?